src/main/resources/db/migration/V1_8__kontor.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE BEHANDLER_DIALOGMELDING_KONTOR
+(
+    id            SERIAL PRIMARY KEY,
+    partner_id    VARCHAR(50)  NOT NULL UNIQUE,
+    her_id        VARCHAR(50),
+    navn          VARCHAR(255),
+    adresse       VARCHAR(255),
+    postnummer    VARCHAR(4),
+    poststed      VARCHAR(255),
+    orgnummer     VARCHAR(9),
+    dialogmelding_enabled BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
Husker ikke i detalj den diskujson vi hadde, men snakket vi ikke om et timestampz her i stedet for en boolean for `dialogmelding_enabled` ?
Og så kan man evt. droppe `update_at`

Reply:
Ja, skal se nærmere på det 👍